### PR TITLE
move fab environment settings to new file and parameterize celery config

### PR DESCRIPTION
**Pull request by snopoke**
Best reviewed per commit

### CODE REVIEW

**Comment by dimagimon** [comment]
:+1: No lint errors found.

**Comment by dimagimon** [comment]
:+1: No lint errors found.

**Comment by snopoke** [comment]
Tested by deploying staging

File: fab/environments.yml [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+  should_migrate: True
+
+  pillow_retry_queue_enabled: True
+
+  new_relic_enabled:
+    - hqdjango0-staging.internal.commcarehq.org
+    - hqdjango1-staging.internal.commcarehq.org
+    - hqdb0-staging.internal.commcarehq.org
+
+  celery_params:
```
Comment (snopoke) [inline fab/environments.yml]: Decreased the concurrency on staging.

**Comment by snopoke** [comment]
@czue

**Pull request merged**